src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
+  border-bottom 1px solid $color-grey
+  .dollar-now-value-label
+    font-size: 12px
+  .dollar-now-value-field
```

Comment:
Yeah, it was suppose to be dollar, because we are showing the current value of the dollar. Do you think it's better to remove the dollar prefix or change for something like `usd`?